runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/config/StreamingEngineComputationConfigFetcher.java

```diff
@@ -157,6 +157,19 @@ private static Optional<StreamingConfigTask> fetchConfigWithRetry(
     }
   }
 
+  private static Optional<ComputationConfig> createComputationConfig(StreamingConfigTask config) {
```

Comment:
can we revert the unrelated change, to keep the diff small and on topic?

Reply:
done